### ISSUE TITLE
Can't install the Dotnet Integration Tool after installing plugin

### ISSUE THREAD

**Issue by davestephens** [JetBrains/teamcity-dotnet-plugin#69]
After installing the newest version of the plugin (SNAPSHOT-20170912130407) onto TeamCity 2017.1, it then asks you to select (or install if you haven't already) the "Dotnet Integration Tool". My TeamCity server doesn't have internet access so I manually uploaded the nuget package as directed.

When selecting this uploaded version as the "Dotnet Integration Tool", in my build log I get:
````
[10:21:12]	Step 3/3: Build (.NET)
[10:21:12]	[Step 3/3] Path "C:\buildagent\tools\TeamCity.Dotnet.Integration.1.0.31-beta\msbuild15\TeamCity.MSBuild.Logger.dll" to MSBuild logger was not found
````

If I look on the build agent, the file it wants is at `C:\buildagent\tools\TeamCity.Dotnet.Integration.1.0.31-beta\build\_common\msbuild15`, which presumably is because TeamCity is unpacking the nupkg like a zip file onto the build agents, rather than "installing" it into the right place. 

Am I doing something wrong here, or is there a bug in the way the nupkg is handled by the plugin? Thanks in advance for any help!

**Comment by NikolayPianikov** [comment]
@davestephens thanx for info, fixed

**Comment by davestephens** [comment]
That's awesome, thank you. 

Any idea when this will be released and available for install?

**Comment by NikolayPianikov** [comment]
I've done, try downloading it now

**Comment by davestephens** [comment]
OK, so I've installed the plugin. It then lets me upload the TeamCity.dotnet.integration nupkg, and it installs and unpacks correctly to build agents. Log colouring also works as expected, so thanks for that!

HOWEVER....:)

After installing, these directories appeared in the build agent tools directory - which also seem like unzipped nuget packages as described earlier in this issue: 

![image](https://user-images.githubusercontent.com/668464/30637841-6e6b116a-9df1-11e7-8684-4740f5f4991f.png)

I also see these in the tools listed in the TeamCity server tools page:

![image](https://user-images.githubusercontent.com/668464/30637865-86fc243a-9df1-11e7-833c-0d3a18cc0acc.png)

If I try and select them as the integration tool in a build configuration, it just defaults back to the "proper" installed version when hitting save.

![image](https://user-images.githubusercontent.com/668464/30638256-af7b7df6-9df2-11e7-949d-50e2f95432d0.png)

So, your fix works for the integration package, but seems to have introduced some extra weirdness during installation.

Thanks in advance!
Dave

**Comment by NikolayPianikov** [comment]
@davestephens did you manually rename .nupkg files before uploading? I mean that the newest version of TeamCity.Dotnet.Integration is 1.0.31-beta

**Comment by NikolayPianikov** [comment]
@davestephens did you manually rename .nupkg files before uploading? I mean that the newest version of TeamCity.Dotnet.Integration is 1.0.31-beta

**Comment by davestephens** [comment]
The **only** integration package I installed was 1.0.31-beta, and that works fine.

The other three appeared after I installed the plugin and restarted TeamCity.

**Comment by NikolayPianikov** [comment]
And what do they contain?

**Comment by NikolayPianikov** [comment]
It is very strange. Could you please remove them all and upload `teamcity.dotnet.integration.1.0.31-beta.nupkg` again?

**Comment by davestephens** [comment]
Each one has a skeleton unzipped nupkg (similar to as I described in the first comment in the issue) but the actual "thing" it is a package of appears to be nuget.exe - so each one is a different nuget version that we have installed on the server....another screenshot :) 

![image](https://user-images.githubusercontent.com/668464/30650650-531e97f4-9e1b-11e7-88fc-2316e6ebaaa7.png)

**Comment by NikolayPianikov** [comment]
@davestephens thanx, I will think :)

**Comment by NikolayPianikov** [comment]
@davestephens could you try installing [this version](https://teamcity.jetbrains.com/viewLog.html?buildId=1175724&tab=artifacts&buildTypeId=TeamCityDotnetCorePluginBuild) and if it is possible I would like to look at file `teamcity-server.log` from your TeamCity build server

**Comment by davestephens** [comment]
No probs, will have to be tomorrow now though. Do you want me to remove the random dotnet integration plugins that appeared after installing the last plugin first?

**Comment by NikolayPianikov** [comment]
Yes please, let's try to make a pure experiment

**Comment by NikolayPianikov** [comment]
I've added new issue into the tracker and done some fixes, and I hope it helps. Fixes are in the `master` branch now and I will make a new release soon. So try [it](https://teamcity.jetbrains.com/viewLog.html?buildId=1176058&buildTypeId=TeamCityDotnetCorePluginBuild&tab=artifacts) and if there are something is wrong let me know please

**Comment by davestephens** [comment]
OK, finally got this installed. Didn't get all the weird tools appear this time...will run it for a bit and get back to you.

**Comment by davestephens** [comment]
Ok, since updating I now see:

````
Starting test execution, please wait...
[12:43:17]	[Step 4/4] Could not find a test logger with URI or FriendlyName 'logger://teamcity'.
````
Previously I'd get the test output to the console.

**Comment by bassebaba** [comment]
Hello. I'm sorry but I don't understand what step I'm missing.
I installed the plugin, did not get asked to do anything else.

When I try to build, this is the result:
```
Failed to start build #4 at 06 Oct 17 12:19 on dotnet-core
.NET integration package was not found at "/opt/buildagent/plugins/dotnet-core-plugin/tools"
```

This is what I installed:
https://plugins.jetbrains.com/plugin/9190--net-cli-support

How do I install the "Dotnet Integration" under tools? It's not selectable in the "+ Install tool" menu.
